free-programming-books.md

```diff
@@ -837,6 +837,7 @@
 * [The Bash Academy](http://www.bash.academy)
 * [The Linux Command Line](http://linuxcommand.org/tlcl.php) - William E. Shotts, Jr. (PDF)
 * [Writing Shell Scripts](http://linuxcommand.org/lc3_writing_shell_scripts.php) - William E. Shotts, Jr.
+* [Bash tutorial](http://gdrcorelec.ups-tlse.fr/files/bash.pdf) (PDF)
```

Comment:
check the [guidlines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#guidelines) for help on passing tests. (alphabetization, include author.)